.ci/scripts/test_export_data_command.sh

```diff
@@ -21,7 +21,7 @@ python -m synapse.app.homeserver --generate-keys -c .ci/sqlite-config.yaml
 echo "--- Prepare test database"
 
 # Make sure the SQLite3 database is using the latest schema and has no pending background update.
-scripts/update_synapse_database --database-config .ci/sqlite-config.yaml --run-background-updates
+update_synapse_database --database-config .ci/sqlite-config.yaml --run-background-updates
```

Comment:
I think this might have a bad rebase in it or something? I think these scripts weren't moved until after we branched?

Reply:
Ahh, I think this was originally branched off from `develop` but not rebased on to the release branch. Let me fix that now.

Reply:
Yeah, the history was:
- branch off develop
- commits
- merge in origin/develop to fix CI
- retarget PR at the release branch

I needed to do a rebase at that last step. That's done now.